packages/ui/src/app/pages/WorkingGroups/MyRoles/MyRole.tsx

```diff
@@ -53,7 +53,8 @@ export const MyRole = () => {
   const workerExcessValue = useMemo(() => {
     if (canMoveExcessTokens) {
       const excessValue = worker.stake.sub(worker.minStake)
-      return stakeBalance.transferable.gt(excessValue) ? new BN(excessValue) : stakeBalance.transferable
+
+      return stakeBalance.transferable.gte(excessValue) ? new BN(excessValue) : stakeBalance.transferable
```

Comment:
@stepien4soft actually please still move the `number` to `BN` casting to `@/bounty/components/BountiesTiles.tsx`.
Once #3285 will be fixed it will become `string` to `BN` so the code won't change.